src/scripts/services/notifications/dimItemNotificationGenerator.factory.js

```diff
@@ -0,0 +1,64 @@
+const angular = require('angular');
+const _ = require('underscore');
+
+(function() {
+  'use strict';
+
+  angular.module('dimApp')
+    .factory('dimItemNotificationGenerator', ItemNotificationGenerator)
+    .run(['dimItemNotificationGenerator', (dimItemNotificationGenerator) => {
+      // Assuming on for prototype
+      dimItemNotificationGenerator.start();
+    }]);
+
+  function ItemNotificationGenerator($rootScope, dimNotificationsService) {
+    const createNotification = dimNotificationsService.registerNotificationGenerator({
+      name: "Item Notification Generator",
+      description: "I tell people when they get shiny new loot."
+    });
+    let cancelReloadListener;
+    const alreadyNotified = [];
+
+    // Ideally we'd start/stop this as the user turns it on/off to avoid unneeded processing.
+    return {
+      active: false,
+      start: () => {
+        var self = this;
+        function checkForNewStuff(store) {
+          if (!store) {
+            return;
+          }
+          const newItems = _.filter(store.items, function(item) { return item.isNew && !item.notransfer && item.equipment && !item.classified && _.indexOf(alreadyNotified, item.id) >= 0; });
```

Comment:
If you want to test without the game running, removing item.isNew from the filter should send you a notification using all your gear.